observing/makesdf.py

```
@@ -40,17 +40,17 @@ def create(out_name, sess_id=None, sess_mode=None, beam_num=None, cal_dir='/home
         beam_num = None
         cal_dir = None
 
-    if pi_id is None:
-        pi_id = random.randint(0, 1000)
-        print(f"No PI ID provided. Setting random PI ID of {pi_id}")
-
     if pi_name is None:
         try:
             pi_name = os.environ["USER"]
         except:
```

I don't like blanket excepts. I think you're looking for a `KeyError` here. Say if someone accidentally removed the `import os`, then all the pi_names will become observer, because the `NameError` gets caught.

That aside, since this is a script that a person runs -- maybe it's a good idea to just throw an error here if pi_name is not specified?